frontend/ui/widget/dictquicklookup.lua

```diff
@@ -794,12 +796,60 @@ function DictQuickLookup:update()
     end
 
     -- Update main text widgets
-    if self.is_html then
+    if self.is_html and self.shw_widget then
         self.text_widget.htmlbox_widget:setContent(self.definition, self:getHtmlDictionaryCss(), Screen:scaleBySize(self.dict_font_size))
-    else
+        -- Scroll back to top
+        self.text_widget.htmlbox_widget.page_number = 1
+        self.text_widget.v_scroll_bar:set((self.text_widget.htmlbox_widget.page_number-1) / self.text_widget.htmlbox_widget.page_count, self.text_widget.htmlbox_widget.page_number / self.text_widget.htmlbox_widget.page_count)
+        self.text_widget.v_scroll_bar.enable = self.text_widget.htmlbox_widget.page_count > 1
```

Comment:
This hacking of the internal (yes, we might already do it elsewhere...) could probably be made a method of both Scroll*Widget:scrollToTop().